mindsdb_native/libs/helpers/confidence_helpers.py

```diff
@@ -115,3 +125,27 @@ def get_categorical_conf(all_confs, conf_candidates):
         else:
             significances.append(0.005)  # default: not confident label is the predicted one
     return significances
+
+
+def get_anomalies(bounds, observed_series, cooldown=1):
```

Comment:
what is the role of cooldown? It looks like when an anomally triggers you reset the counter

Reply:
Good question, I'll add this in the code too.

Cooldown is a grace period during which the predictor will be coping with the dynamics introduced by the anomaly, so we avoid additional flagging in case the series has gone back to its normal dynamics but the predictor hasn't yet. This is useful for spike-type anomalies, maybe not so for gradual ones.

Reply:
does this require the user to have a notion of whether they have spike or gradual anomalies? (i.e. maybe like a detector - a spike can be a 'recognized signal' versus a gradual one a temperature signal being a slow loss of temperature over time)

Reply:
Yup! And this being a first iteration, we will have to explore how to best tackle gradual anomalies.